portability-spi-cloud/src/main/java/org/dataportabilityproject/spi/cloud/types/PortabilityJob.java

```diff
@@ -37,6 +39,9 @@
     @JsonProperty
     private LocalDateTime lastUpdateTimestamp; // ISO 8601 timestamp
 
+    @JsonProperty
+    private JobAuthorization jobAuthorization; // Review: Job Authorization data
```

Comment:
Was that Review comment a note for yourself to come back to? Not sure what it means